react/components/AccordionFilterItem.js

```diff
@@ -1,12 +1,12 @@
-import React, { Fragment } from 'react'
-import PropTypes from 'prop-types'
-import { intlShape, injectIntl } from 'react-intl'
 import classNames from 'classnames'
-import { Tag } from 'vtex.styleguide'
-import { useCssHandles } from 'vtex.css-handles'
+import PropTypes from 'prop-types'
+import React, { Fragment } from 'react'
+import { injectIntl, intlShape } from 'react-intl'
+import { applyModifiers, useCssHandles } from 'vtex.css-handles'
 import { IconCaret } from 'vtex.store-icons'
-
+import { Tag } from 'vtex.styleguide'
 import { getFilterTitle } from '../constants/SearchHelpers'
+import { Slugify } from './FilterNavigator/legacy/hooks/useSelectedFilters'
```

Comment:
This change is not necessary, no need to reorder the imports.
Also, it's good practice to always have the import for `react` as the first one.